### ISSUE TITLE
Jobs Failing on travis-ci for ppc64le architecture

### ISSUE THREAD

**Issue by kishorkunal-raj** [bsm/redeo#25]
Hi All,
I am using Travis-CI for running jobs on ppc64le architecture but the jobs failed and exited with status 2. Please help me out . Logs are mentioned below

logs
========
Setting environment variables from .travis.yml

$ export GO111MODULE=on

0.10s$ travis_setup_go

go version go1.11.13 linux/ppc64le

$ export GOPATH="/home/travis/gopath"

$ export PATH="/home/travis/gopath/bin:/home/travis/.gimme/versions/go1.11.13.linux.ppc64le/bin:/home/travis/bin:/home/travis/bin:/home/travis/.local/bin:/home/travis/.rvm/gems/ruby-2.7.0/bin:/home/travis/.rvm/gems/ruby-2.7.0@global/bin:/home/travis/.rvm/rubies/ruby-2.7.0/bin:/home/travis/.phpenv/shims:/home/travis/perl5/perlbrew/bin:/home/travis/.nvm/versions/node/v8.12.0/bin:/home/travis/.kiex/elixirs/elixir-1.9.1/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/home/travis/.phpenv/bin:/home/travis/.rvm/bin"

$ export GO111MODULE="on"

$ gimme version

v1.5.4

$ go version

go version go1.11.13 linux/ppc64le
go.env

$ go env
install

12.76s$ go get -u github.com/go-redis/redis

248.86s$ make default fuzz fuzzrace

go vet ./...

go: downloading github.com/bsm/pool v0.8.1

go: downloading github.com/onsi/gomega v1.5.0

go: downloading github.com/onsi/ginkgo v1.8.0

go: downloading gopkg.in/yaml.v2 v2.2.2

go: downloading golang.org/x/net v0.0.0-20190724013045-ca1201d0de80

go: downloading github.com/hpcloud/tail v1.0.0

go: downloading gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7

go: downloading gopkg.in/fsnotify.v1 v1.4.7

go: downloading golang.org/x/sys v0.0.0-20190812073006-9eafafc0a87e

go: downloading golang.org/x/text v0.3.2

go test ./...

ok  	github.com/bsm/redeo	0.113s

ok  	github.com/bsm/redeo/client	0.007s

?   	github.com/bsm/redeo/cmd/redeo-server-example	[no test files]

ok  	github.com/bsm/redeo/info	0.007s

ok  	github.com/bsm/redeo/redeotest	0.013s

ok  	github.com/bsm/redeo/resp	0.047s

go test ./.fuzz

ok  	github.com/bsm/redeo/.fuzz	20.381s

go test -race ./.fuzz

==================

WARNING: DATA RACE

Write at 0x00c000102151 by goroutine 13:

  testing.(*common).FailNow()

      /home/travis/.gimme/versions/go1.11.13.linux.ppc64le/src/testing/testing.go:589 +0x40

  testing.(*common).Fatalf()

      /home/travis/.gimme/versions/go1.11.13.linux.ppc64le/src/testing/testing.go:634 +0x74

  github.com/bsm/redeo/%2efuzz_test.fuzzIteration()

      /home/travis/gopath/src/github.com/kishorkunal-raj/redeo/.fuzz/fuzz_test.go:59 +0xf8

  github.com/bsm/redeo/%2efuzz_test.TestFuzz.func1()

      /home/travis/gopath/src/github.com/kishorkunal-raj/redeo/.fuzz/fuzz_test.go:48 +0x94

Previous write at 0x00c000102151 by goroutine 23:

  testing.(*common).FailNow()

      /home/travis/.gimme/versions/go1.11.13.linux.ppc64le/src/testing/testing.go:589 +0x40

  testing.(*common).Fatalf()

      /home/travis/.gimme/versions/go1.11.13.linux.ppc64le/src/testing/testing.go:634 +0x74

  github.com/bsm/redeo/%2efuzz_test.fuzzIteration()

      /home/travis/gopath/src/github.com/kishorkunal-raj/redeo/.fuzz/fuzz_test.go:90 +0x414

  github.com/bsm/redeo/%2efuzz_test.TestFuzz.func1()

      /home/travis/gopath/src/github.com/kishorkunal-raj/redeo/.fuzz/fuzz_test.go:48 +0x94

Goroutine 13 (running) created at:

  github.com/bsm/redeo/%2efuzz_test.TestFuzz()

      /home/travis/gopath/src/github.com/kishorkunal-raj/redeo/.fuzz/fuzz_test.go:44 +0x3bc

  testing.tRunner()

      /home/travis/.gimme/versions/go1.11.13.linux.ppc64le/src/testing/testing.go:827 +0x140

Goroutine 23 (finished) created at:

  github.com/bsm/redeo/%2efuzz_test.TestFuzz()

      /home/travis/gopath/src/github.com/kishorkunal-raj/redeo/.fuzz/fuzz_test.go:44 +0x3bc

  testing.tRunner()

      /home/travis/.gimme/versions/go1.11.13.linux.ppc64le/src/testing/testing.go:827 +0x140

==================

--- FAIL: TestFuzz (199.51s)

    fuzz_test.go:90: fuzzstream failed with: ERR unknown command '�a�s:���Թ7k�^�w�v���O�~6���Z����::`�tl_�72<�I`�\�N��G��g*����@��³�3�����L���', command: "fuzzstream: ERR unknown command '\x84\x03a\xabs:\u007f\xd4\xf8\xb8Թ7\x19k\xc9^\xdbw\xd6v\xe9\xf5\xe8O\xbe~\x196\xee\xeb\xabZ\xa3\xfa\x9e\x93::`\xfdtl_\x19\xc972<\x95I`\x03\xef\\\xbdN\x91\xccG\x17\x97\xefg*\xed\xf6\xb7\xfc@\x0e\x16\x80\x8d³\x903\x94\xe2\u007f\xd9\xed\xfdL\xbe\x97\x00\xb3'"

    fuzz_test.go:59: fuzzmb failed with: ERR unknown command '�{��N���CD{u����r��X�vd.��6\q�|ߛ�h�', command: "fuzzmb [210 239 118 74 138 61 89 249 118 41 186
..
..
..
\x14\x97'"

    testing.go:771: race detected during execution of test

FAIL

FAIL	github.com/bsm/redeo/.fuzz	199.531s

make: *** [fuzzrace] Error 1

Makefile:16: recipe for target 'fuzzrace' failed

The command "make default fuzz fuzzrace" exited with 2.

Done. Your build exited with 1.

Please excuse if I asked irrelevant as am newbie here

**Comment by dim** [comment]
Sorry, it's hard to follow the logs since they are not formatted. By the looks of it, it seemed like a data race in the testing package, which could be a big from the past. I suggest you change the Travis config to test with the latest two Go versions (which is what Google currently supports), i.e. 1.14 + 1.15 only
